### DISCUSSION TITLE
Check if a plugin was added

### DISCUSSION THREAD

**Discussion by nuzzles** [bevyengine/bevy#16153] [Q&A] [answered]
How can I see if a Plugin was added?

**Comment by BD103** [comment, marked as answer]
[`App::is_plugin_added()`](https://docs.rs/bevy/latest/bevy/app/struct.App.html#method.is_plugin_added) should be what you're looking for!

```rust
let mut app = App::new();
app.add_plugins(MyPlugin);
assert!(app.is_plugin_added::<MyPlugin>());
```
